app/views/discovery_rules/_form.html.erb

```diff
@@ -6,7 +6,18 @@
                      :control_group_id => 'search_group', :size => 'col-md-9' %>
   <%= select_f f, :hostgroup_id, accessible_hostgroups, :id, :to_label, { :include_blank => false },
     { :help_inline => _('Target host group for this rule with all properties set')} %>
-  <%= text_f f, :hostname, :help_inline => _('Target hostname template pattern (empty for MAC)')  %>
+  <%= text_f f, :hostname, :help_inline => popover(_("Target hostname template pattern"),_("<dl>" +
+                                                                  "<dd>Domain will be appended automatically.</dd>" +
```

Comment:
We can't translate like this, otherwise translators will mess this up. http://projects.theforeman.org/projects/foreman/wiki/Translating

The best approach here is something like:

    "<dl>" +
    "<dd>" + _("Domain will be appended automatically.") + "</dd>" +
    ...